spec/models/work_packages/scopes/relatable_spec.rb

```diff
@@ -27,6 +27,8 @@
 require 'spec_helper'
 
 describe WorkPackages::Scopes::Relatable, '.relatable scope' do
+  create_shared_association_defaults_for_work_package_factory
+
   let(:project) { create(:project) }
```

Comment:
This change:
```suggestion
  let(:project) { project_with_types }
```

makes the spec time go from 16 seconds to 8.5 seconds as it reuses the project created in a `shared_let` when calling `create_shared_association_defaults_for_work_package_factory`.

Reply:
Makes total sense. But I'll just try to remove all the references to `project` since that should be handled by the `create_shared...` unless I am mistaken and will trim down the spec.

Reply:
yes, that's even better